### PR TITLE
 Python script to generate a service account signed JWT

### PR DESCRIPTION
**Pull request by liminw**

### CODE REVIEW

**Comment by mandarjog** [comment]
We should ensure that 
1. all python files have been formatted with autopep8    (pip install autopep8)
2. An accompanied file_requirements.txt that lists packages needed by the script.

File: security/tools/jwt/sa-jwt.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  if args.issuer:
+    payload["iss"] = args.issuer
+    payload["sub"] = args.issuer
+  else:
+    payload["iss"] = credentials.service_account_email
+    payload["sub"] = credentials.service_account_email
+  signed_jwt = oauth2client.crypt.make_signed_jwt(
+    credentials._signer, payload, key_id=credentials._private_key_id)
+  return signed_jwt
+if __name__ == '__main__':
```
Comment (diemtvu) [inline security/tools/jwt/sa-jwt.py]: nit Can you add 2 blank lines

File: security/tools/jwt/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+It outputs the following:
+```
+usage: sa-jwt.py [-h] [-iss ISSUER] aud service_account_file
+
+Python script generates a JWT signed by a Google service account
+
+positional arguments:
+  aud                   Audience.
```
Comment (diemtvu) [inline security/tools/jwt/README.md]: Can this be optional too? And comma separated for multi entries (not important if it adds to much complexity)

File: security/tools/jwt/sa-jwt.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    # MAX_TOKEN_LIFETIME_SECS is set to one hour by default.
+    "exp": now + credentials.MAX_TOKEN_LIFETIME_SECS,
+    "iat": now,
+    "aud": args.aud,
+    # Add any custom claims here.
+    # e.g.,
+    # "email": alice@yahoo.com
+  }
+  if args.issuer:
+    payload["iss"] = args.issuer
```
Comment (diemtvu) [inline security/tools/jwt/sa-jwt.py]: should we have separate flags for them?

File: security/tools/jwt/sa-jwt.py (hunk 1 above)
Comment (liminw) [reply security/tools/jwt/sa-jwt.py]: Done.

File: security/tools/jwt/README.md (hunk 2 above)
Comment (liminw) [reply security/tools/jwt/README.md]: Made aud claim optional.

File: security/tools/jwt/sa-jwt.py (hunk 3 above)
Comment (liminw) [reply security/tools/jwt/sa-jwt.py]: I added a separate flag for "sub".

**Pull request merged**